setup.py

```diff
@@ -68,6 +68,8 @@
             "six>=1.10.0",
             "lifelines>=0.9.1.0",
             "scikit-learn>=0.17.1",
+            "isovar==0.0.3",
```

Comment:
ah, that explains the failing CI tasks. I think I still have this because of a conflict error with the recent master merge. Will remove it.